lib/screen/home/home_page.dart

```diff
@@ -607,6 +632,30 @@ class HomePageState extends State<HomePage>
             data: {"title": "create_your_first_playlist".tr()});
       }
     }
+
+    final remindTime = configurationService.getShowBackupSettingTip();
+    final shouldRemindNow = remindTime == null || now.isAfter(remindTime);
+    if (shouldRemindNow) {
+      configurationService
+          .setShowBackupSettingTip(now.add(const Duration(days: 7)));
+      bool showTip = false;
+      if (Platform.isAndroid) {
+        final isAndroidEndToEndEncryptionAvailable =
+            await injector<AccountService>()
+                .isAndroidEndToEndEncryptionAvailable();
+        if (isAndroidEndToEndEncryptionAvailable == null) {
+          showTip = true;
+        }
+      } else {
+        final iCloudAvailable = injector<CloudService>().isAvailableNotifier;
+        showTip = !iCloudAvailable.value;
+      }
+      if (showTip) {
```

Comment:
I think here we should add more condition `configurationService.showBackupSettingTip.value == false`, then metric will send only one time when the value is changed.